src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy

```diff
@@ -113,7 +113,7 @@ class DockerCreateContainer extends AbstractDockerRemoteApiTask {
 
     @Input
     @Optional
-    Map<String, Integer> exposedPorts
+    Map<Integer, String> exposedPorts
```

Comment:
There is very problematic to define in configuration which ports should be exposed cause it need Multimap (in case when you expose multiple TCP ports). And Groovy/Gradle  doesn't have multi map implementation in class path.